src/components/AssignAssetComponent.jsx

```diff
@@ -12,11 +12,14 @@ const userEmailsOptions = usersList => usersList.map((typeOption, index) => ({
 
 const AssignedTo = (props) => {
   if (isEmpty(props.assignedUser)) {
+    const hasError = props.errorMessage !== '';
```

Comment:
An empty string will always be a "falsy" value whereas any string behaves as a truthy value. Therefore, the logic you have is the same as:
```js
const hasError = !!props.errorMessage
```
Use `!props.errorMessage` if checking for empty string and `!!props.errorMessage` to coerce a string to a boolean value, instead of comparing to `''`. Therefore, the above can be written as:
```js
const hasError = !!props.errorMessage;